### PR TITLE
Review handling of `docs/.contentlayer`

### PR DESCRIPTION
**Pull request by koralle**
## Description

`docs/.contentlayer` should not be managed by git and should be excluded from the TypeScript compiler's compilation (and type-checking) targets

### CODE REVIEW

File: docs/tsconfig.json [hunk 1]
```diff
@@ -33,5 +33,5 @@
     "server.ts",
     "./.mdx"
   ],
-  "exclude": ["node_modules"]
+  "exclude": ["node_modules", ".contentlayer"]
```
Comment (Tomoya-Matsubara) [inline docs/tsconfig.json:36]: I am not very familiar with `Contentlayer`, but the [`Contentlayer` documentation](https://contentlayer.dev/docs/getting-started-cddd76b7#typescript-configuration) suggests including `.contentlayer/generated` in `tsconfig.json`.

Should `.contentlayer` be really excluded?

Comment (koralle) [reply docs/tsconfig.json:36]: I did it haphazardly. I'll fix it.

**Comment by hirotomoyamada** [comment]
@koralle 

Why did you create this PR when you discontinued the `contentlayer`?

**Comment by koralle** [comment]
> @koralle 
> 
> Why did you create this PR when you discontinued the `contentlayer`?

**Pull request closed without merging**